web/templates/setup/index.html.eex

```diff
@@ -0,0 +1,92 @@
+<div class="hero">
+  <img class="hero__logo" src="/images/post_logo.png" alt="POST logo - Meet POST Bot"/>
+  <h1 class="hero__text">Meet POST Bot &ndash;</br>
+  the easiest way to get valuable feedback on your Articles</h1>
+  <%= link to: auth_path(@conn, :request, :slack) do %>
+    <img
+      alt="Sign in with Slack"
+      height="40"
+      width="172"
+      src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
+      srcset="https://platform.slack-edge.com/img/sign_in_with_slack.png 1x, https://platform.slack-edge.com/img/sign_in_with_slack@2x.png 2x"/>
+  <% end %>
+</div>
+
+<div class="bottom">
+  <div class="bottom__offset container">
+    <h2 class="bottom__title">Welcome to POST Bot:</h2>
+    <ul class="steps steps--setup">
+      <li class="steps__item">
+        <div class="steps__left">
+          <%= if @current_team && @current_team.google_token do %>
```

Comment:
If the user has been authenticated via slack, `@current_team` will be present, so just `if @current_team` should do it